static/app/views/dashboardsV2/widgetBuilder/utils.tsx

```diff
@@ -187,3 +188,8 @@ export function getParsedDefaultWidgetQuery(query = ''): WidgetQuery | undefined
     aggregates,
   } as WidgetQuery;
 }
+
+export function getFields(fieldsString: string): string[] {
+  // Use a negative lookahead to avoid splitting on commas inside equation fields
+  return fieldsString.split(/,(?![^(]*\))/g);
```

Comment:
I was testing this in Safari because I had a feeling that negative lookaheads were not working there. (this is what I was remembering: https://caniuse.com/js-regexp-lookbehind)

While testing there I noticed that the widget library does not look great:
![image](https://user-images.githubusercontent.com/9060071/157197612-70a1437c-7d6d-4c45-9695-140d454b313a.png)

Reply:
Thanks for finding that. We have some queued up work to make the sidebar a bit larger as well so we'll keep this in mind!